core/src/main/java/io/questdb/PropServerConfiguration.java

```diff
@@ -563,7 +563,7 @@ public PropServerConfiguration(
         // changing the default value of walEnabledDefault to true would mean that QuestDB instances upgraded from
         // a pre-WAL version suddenly would start to create WAL tables by default, this could come as a surprise to users
         // instead cairo.wal.enabled.default=true is added to the config, so only new QuestDB installations have WAL enabled by default
-        this.walEnabledDefault = getBoolean(properties, env, PropertyKey.CAIRO_WAL_ENABLED_DEFAULT, false);
+        this.walEnabledDefault = getBoolean(properties, env, PropertyKey.CAIRO_WAL_ENABLED_DEFAULT, true);
```

Comment:
With this change, we can comment out the prop in `server.conf`:
https://github.com/questdb/questdb/blob/af4ca60a7ab3ff6b9879d9b4ebc675ab45c5edc3/core/src/main/resources/io/questdb/site/conf/server.conf#L687
https://github.com/questdb/questdb/blob/af4ca60a7ab3ff6b9879d9b4ebc675ab45c5edc3/pkg/ami/marketplace/assets/server.conf#L649